text/0001-cdk-update.md

```diff
@@ -269,10 +269,27 @@ off of information in the CDK `App` context.
 ### What is the high level implementation plan?
 
 We will start from the prototype CDK update command that identifies the Lambda
-resources and then publishes them using the CDK toolkit.
-
-We will propose updates to the initial CDK constructs that are capable of
-accelerated deployment and make those changes to the CDK.
+resources and then publishes them using the CDK toolkit, and extend that to
```

Comment:
Rename "toolkit" with "CLI" please (we no longer use this term)

Reply:
Shall do